### PR TITLE
[NETBEANS-4805] Fixed Naming suggestions of Record variable/identifier

### PR DESCRIPTION
**Pull request by singh-akhilesh**
https://issues.apache.org/jira/browse/NETBEANS-4805

### CODE REVIEW

**Review by arusinha — APPROVED** [review]

**Comment by matthiasblaesing** [comment]
@arusinha please merge if this is good to go. @singh-akhilesh is not a committer, so can't merge himself.

File: java/java.completion/src/org/netbeans/modules/java/completion/JavaCompletionTask.java [hunk 1]
```diff
@@ -6267,6 +6267,9 @@ private static ElementKind simplifyElementKind(ElementKind kind) {
         if (TreeShims.BINDING_VARIABLE.equals(kind.name())) {
             return ElementKind.LOCAL_VARIABLE;
         }
+        else if(TreeShims.RECORD.equals(kind.name())){
+            return ElementKind.CLASS;
+        }
```
Comment (junichi11) [inline java/java.completion/src/org/netbeans/modules/java/completion/JavaCompletionTask.java:6272]: Should format (select 6269-6272).

Comment (singh-akhilesh) [reply java/java.completion/src/org/netbeans/modules/java/completion/JavaCompletionTask.java]: comments addressed

**Comment by lkishalmi** [comment]
Seems file to me. Merging.

**Comment by arvindaprameya** [comment]
Approved

**Pull request merged**